org.gnome.chess.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.gnome.chess",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "gnome-chess",
+    "copy-icon": true,
+    "rename-icon": "gnome-chess",
+    "rename-desktop-file": "gnome-chess.desktop",
+    "rename-appdata-file": "gnome-chess.appdata.xml",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* dconf */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        /* Documents directory access for storing saved games */
+        "--filesystem=xdg-documents"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup": ["/share/gnuchess", "/share/info", "/share/man"],
```

Comment:
At least `/include` will need to be cleaned up too.

Reply:
Will add